tf2onnx/onnx_opset/controlflow.py

```diff
@@ -289,6 +290,20 @@ def version_7(cls, ctx, node, **kwargs):
         ctx.insert_new_node_on_input(node, "Unsqueeze", node.input[1], name=node.child_name(), axes=[0])
         ctx.insert_new_node_on_output("Squeeze", node.output[0], name=node.child_name(), axes=[0])
 
+    @classmethod
+    def version_13(cls, ctx, node, **kwargs):
```

Comment:
Can you make all versions use this code?  Looks the same as the version_7 handler.

Reply:
Not fixed yet: I need to create a function insert_node_on_input as I did with insert_new_node_on_output